resources/views/layouts/app.blade.php

```diff
@@ -29,6 +29,7 @@ function gtag(){dataLayer.push(arguments);}
 
     <!-- Scripts -->
     <script src="{{ mix('js/app.js') }}" defer></script>
+    <script src="https://kit.fontawesome.com/1aa7dc87ba.js" crossorigin="anonymous"></script>
```

Comment:
please add your needed dependency to the package.json.

Reply:
I've actually removed the dependency all together and just stuck with HTML symbols. I think it's overkill to bring in Font Awesome just for 2 icons. Plus, their new license model means the key was tied to my email address. 

So the result is the same, but no extra dependency needed 😄